libs/types/src/election.ts

```diff
@@ -112,6 +112,10 @@ export interface Candidate {
   readonly partyIds?: readonly PartyId[];
   readonly isWriteIn?: boolean;
   readonly writeInIndex?: number;
+  // Structured name properties are supported only in VxDesign.
+  readonly firstName?: string;
+  readonly middleName?: string;
+  readonly lastName?: string;
```

Comment:
I thought about extending `Candidate` to a type used only in VxDesign eg. `VxDesignCandidate` found that muddied the existing `CandidateContest` type. We'd have to loosen `CandidateContest.candidates` to allow `Candidate[] | VxDesignCandidate[]` which IMO causes a lot more headache downstream in other VX apps than just having these 3 optional fields.

Reply:
Makes sense for now, for expediency sake.

After looking through the VxDesign code a bit though, it does feel like it's outgrowing the data types we use in the rest of the product (and we've already started splitting out a few VxDesign-specific types like ballot styles and precinct splits), so could make sense, longer term, to represent stuff like this in separate types for VxDesign, which would then handle translating those into the election package format when exporting. Then we also wouldn't need to expose unused fields to downstream consumers and reduce confusion.